tutorials/physics/rigid_body.rst

```diff
@@ -0,0 +1,51 @@
+.. _doc_rigid_body
+
+Rigid Body
+=========
+
+What is a rigid body?
+--------------------
+
+Rigid body is a rigid (non deformable) physical object that is directly controlled by physics engine in order to simulate phisical behaviour of objects.
```

Comment:
You shouldn't define a term with the same term. It's not a physical object, it's a *simulation* of one.

A rigid body is one that is directly controlled by the physics engine in order to simulate the behavior of physical objects.